rust/routee-compass-core/src/model/traversal/default/speed_traversal_model.rs

```diff
@@ -104,23 +111,35 @@ impl TraversalModel for SpeedTraversalModel {
         if distance == Distance::ZERO {
             return Ok(());
         }
+        // <<<<<<< HEAD
```

Comment:
Looks like there are some relics of the merge left over here (unless you had intended to leave them in?)

Reply:
SAY WHAAAAAT????????? 😱